src/job-exporter/deploy/job-exporter.yaml.template

```diff
@@ -102,6 +116,14 @@ spec:
         - name: gpu-config
           configMap:
             name: gpu-configuration
+        {%- if cluster_cfg['cluster']['common']['deploy-in-aks'] == "true" %}
+        - name: usr-bin
+          hostPath:
+            path: /usr/bin
+        - name: usr-local-nvidia-bin
+          hostPath:
+            path: /usr/local/nvidia/bin
```

Comment:
mount `/usr/local/nvidia`?

Reply:
Fixed

---
In reply to: [336347682](https://github.com/microsoft/pai/pull/3749#discussion_r336347682) [](ancestors = 336347682)